server/models/AtMode.js

```diff
@@ -21,14 +21,17 @@ module.exports = function (sequelize, DataTypes) {
             },
             name: {
                 type: DataTypes.TEXT,
-                // type: DataTypes.ENUM(
-                //     MODE.READING,
-                //     MODE.INTERACTION,
-                //     MODE.MODELESS
-                // ),
                 defaultValue: MODE.MODELESS,
```

Comment:
I thought I saw references somewhere that implied VoiceOver's quick nav was akin to a mode switcher, like I think I saw some "mode instructions" for VoiceOver that say to turn quick nav off ... but it sounds like we're still treating VoiceOver as modeless? I was wondering if you could bring me up to speed on what we decided about that?

Reply:
> I thought I saw references somewhere that implied VoiceOver's quick nav was akin to a mode switcher

I'm not too sure on that statement.

> I was wondering if you could bring me up to speed on what we decided about that?

VO can have a 'quick nav on' mode, 'quick nav off' mode and any AT could technically support a modeless state (but right now, only VO's v2 tests show any examples of that).